src/Component/BookingTable/index.js

```diff
@@ -8,16 +8,51 @@ import TextField from '@mui/material/TextField';
 import MainButton from '../MainButton'
 import AppBar from '@mui/material/AppBar';
 import Button from '@mui/material/Button';
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { useContext } from "react";
+import ProductContext from "../Context";
+import { Link } from 'react-router-dom';
 function BookingTable() {
+    function validate() {
+        if (document.getElementById('filled-requireda').value == '') {
+            alert("Please Provide Details!");
+            document.getElementById('filled-requireda').focus();
+            return false;
+        }
+        else if (document.getElementById('filled-requiredb').value == '') {
+            alert("Please Provide Name!");
+            document.getElementById('filled-requiredb').focus();
+            return false;
+        }
+        else
+            return true;
+    }
```

Comment:
We need to implement validation in a different way, so we can show error messages instead of alert
something like the code snippet below:
`import React, { useRef, useState } from 'react';

function MyForm() {
  const inputARef = useRef(null);
  const inputBRef = useRef(null);
  const [errorMessage, setErrorMessage] = useState('');

  function validate() {
    if (inputARef.current.value === '') {
      setErrorMessage('Please provide details!');
      inputARef.current.focus();
      return false;
    } else if (inputBRef.current.value === '') {
      setErrorMessage('Please provide a name!');
      inputBRef.current.focus();
      return false;
    } else {
      setErrorMessage('');
      return true;
    }
  }

  return (
    <div>
      <input type="text" ref={inputARef} id="filled-requireda" />
      <input type="text" ref={inputBRef} id="filled-requiredb" />
      {errorMessage && <p>{errorMessage}</p>}
      <button onClick={validate}>Submit</button>
    </div>
  );
}

export default MyForm;
`